index.js

```diff
@@ -60,19 +60,40 @@ const Resizer = ({ array, updateCallback }) => {
   `;
 };
 
+let mouseDown = false;
+let dragType = 1;
+
 const Input = ({ array, updateCallback }) => {
-  const updateCell = (rowIndex, colIndex) => () => {
+  const updateCell = (rowIndex, colIndex) => {
     array[rowIndex][colIndex] ^= 1;
     updateCallback(array);
   };
 
+  const updateCellDrag = (rowIndex, colIndex) => {
```

Comment:
We don't need a second method - why not just use the one above, which flips the cell being dragged over?
This also removes the need for the `dragType` variable.

Reply:
If you go over a cell that you've already gone over in the same drag, then it gets flipped again. This is probably not user-friendly.